tests/test_config_helper_tools.py

```diff
@@ -187,3 +216,9 @@ def test_update_update_waveforms(config0):
     assert config["waveforms"]["flux_line_bias_single_wf"]["samples"] == list(
         gaussian(112, 20)
     )
+
+
+def test_transform_negative_delays(config1):
```

Comment:
A test must have asserts, not prints.
I would add a manually corrected config, and then add two tests:

1. Check that config after function matches the manually corrected config
2. Check that if "create_new_config=True", then the original one did not change, and that the new one matches the manually corrected config.

Reply:
Done :)